documentation/staging/content/release-notes.md

```diff
@@ -6,52 +6,104 @@ draft: false
+* Cluster resource: the operator now provides a new custom resource, Cluster, which configures a specific WebLogic cluster. This resource can be used with Kubernetes Horizontal Pod Autoscaler (HPA) or similar technologies to scale WebLogic clusters.
+* Status and Events: the status of and the Events generated about Domain and Cluster resources have been significantly updated. The operator more clearly communicates when the domain or individual clusters have reached the intended state or significant availability for your application workloads. Failures are distinguished between those that require customer intervention or those that are potentially temporary and will be retried.
+* Istio-compatibility: domain configurations are always compatible with Istio and other service mesh products. Customers are no longer required to enable this support.
+
+##### Upgrade Notes
+
+* This release introduces a new API version for the Domain resource, `weblogic.oracle/v9`. The previous API version, `weblogic.oracle/v8` introduced with Operator 3.0.0 is deprecated, but is still supported.
+* The operator provides a [schema conversion webhook]({{< relref "/managing-operators/conversion-webhook.md" >}}) that automates the upgrade of `weblogic.oracle/v8` Domain resources to `weblogic.oracle/v9`. Customers do not need to take any action to begin using the latest Domain schema; however, the team recommends starting with "v9" Domain resources for any new projects. All samples, tools, and documentation have been updated to use "v9". The operator provides a library that can be used to update existing "v8" Domain YAML files to "v9".
+* This release also introduces a new custom resource, the `weblogic.oracle/v1` Cluster resource. This Cluster resource configures a specific WebLogic cluster and allows customers to scale that cluster using the Kubernetes Horizontal Pod Autoscaler (HPA) or similar technologies. The Cluster resource schema is similar to the former `weblogic.oracle/v8` Domain resource `.spec.clusters[*]` content.
+* Customers who install a single WebLogic Kubernetes Operator per Kubernetes cluster (most common use case) can upgrade directly from any 3.x operator release to 4.0.0. The Helm chart for 4.0.0 will automatically install the schema conversion webhook.
+* For those customers who install more than one WebLogic Kubernetes Operator in a single Kubernetes cluster, you must upgrade every operator to at least version 3.4.1 before upgrading any operator to 4.0.0. As the 4.0.0 Helm chart now also installs a schema conversion webhook, you will want to use the Helm chart option to install this webhook in its own namespace prior to installing any of the 4.0.0 operators.
+* Several Helm chart default values have been changed. Customers who upgrade their 3.x installations using the `--reuse-values` option during the Helm upgrade will continue to use the values from their original installation.
+
+##### Changes by Kind
```

Comment:
Instead of putting all in one section, maybe re-org a little and break this into three titled sections titled "Changes to domain schema" and "Changes to operator Helm chart" and "Minor features and fixes".